resources/views/shopOwner/index.blade.php

```diff
@@ -8,7 +8,7 @@
               <b>Total Orders</b>
             </div>
             <div class="card-body text-center">
-              <h4 class="card-text">99</h4>
+              <h4 class="card-text">{{$orderCount->count()}}</h4>
```

Comment:
Why we run this count on the blade? This should be just rendered.

Reply:
for total order i can count it on controller but for displaying the latest orders down there, we need to loop inside it and get the latest orders list. so it has to be this way otherwise it shows error. 
Anyway, it works fine